docs/src/app/(public)/(content)/react/overview/releases/page.mdx

```diff
@@ -3,6 +3,215 @@
+  The CSS variables are now on the `<ScrollArea.Viewport>` part, not `<ScrollArea.Root>`, which is where they'll typically be used to reduce how many inherit CSS properties need to be specified (for example: on pseudo-elements), and as stated, children no longer inherit the variables.
+  ([#3156](https://github.com/mui/base-ui/pull/3156))
+
+### Select
+
+- **Breaking change:** Make the trigger native button by default.<br />
+  The trigger will now render a `<button>` element, be sure to adjust your code if necessary.
+  ([#3177](https://github.com/mui/base-ui/pull/3177))
+- Add `open` state type on `Select.Icon` interface ([#2919](https://github.com/mui/base-ui/pull/2919))
+- Fix `onValueChange` type inference when `value` is unspecified ([#2897](https://github.com/mui/base-ui/pull/2897))
+- Fix `required` form submission with multiple values ([#2925](https://github.com/mui/base-ui/pull/2925))
+- Avoid re-rendering on popup height expansion ([#2972](https://github.com/mui/base-ui/pull/2972))
+- Place `overflow: hidden` on `<body>` for overlay scrollbars by default. Avoids sticky elements shifting if `<body>` has an `overflow` style specified. ([#3083](https://github.com/mui/base-ui/pull/3083))
+- Add `data-placeholder` attribute ([#2737](https://github.com/mui/base-ui/pull/2737))
+
+### Slider
+
+- **Breaking change:** Add `thumbCollisionBehavior` prop.<br />
+  When multiple thumbs overlap, moving them with mouse will now push the other thumbs to avoid collisions.
+  Other prop options include `swap` and `none` (the current behavior).
+  ([#2856](https://github.com/mui/base-ui/pull/2856))
+- Granular change reasons ([#3132](https://github.com/mui/base-ui/pull/3132))
+
+### Tabs
+
+- **Breaking change:** Fix selected/active state naming consistency.<br />
+  - Renamed `[data-selected]` to `[data-active]` in `<Tabs.Tab>`
+  - Removed `[data-highlighted]` (`:focus-visible` was already the recommendation in styles)
+  - `selectedTabPosition`/`selectedTabSize` are now `activeTabPosition`/`activeTabSize` in `Tabs.Indicator.State`
+    ([#3024](https://github.com/mui/base-ui/pull/3024))
+- **Breaking change:** Change `activateOnFocus` to false.<br />
+  If you need your Tabs to activate on focus, be sure to add `activateOnFocus` prop.
+  ([#3176](https://github.com/mui/base-ui/pull/3176))
+- Fix Next.js 16 error from `Math.random` id generation ([#3051](https://github.com/mui/base-ui/pull/3051))
+
+### Toast
+
+- Allow `React.ReactNode` for `title`/`description` properties ([#2929](https://github.com/mui/base-ui/pull/2929))
+- Add ability to anchor to an element ([#3096](https://github.com/mui/base-ui/pull/3096))
+
+### Toolbar
+
+- **Breaking change:** Drop `cols` prop.
+
+  This prop was not supposed to be exposed.
+  ([#3133](https://github.com/mui/base-ui/pull/3133))
+
+### Tooltip
+
+- **Breaking change:** Support detached triggers.
+
+  TODO: DESCRIBE THE BREAKING CHANGE.
+  ([#3071](https://github.com/mui/base-ui/pull/3071))
```

Comment:
The remaining TODOs seem to just be about moving the delay props to the trigger?